src/hotspot/share/jfr/metadata/metadata.xml

```diff
@@ -1091,8 +1091,8 @@
   </Event>
 
   <Event name="GCLocker" category="Java Virtual Machine, GC, Detailed" description="GC Locker Information" label="GC Locker" startTime="true" thread="true" stackTrace="true">
-    <Field type="uint" name="lockCount" label="Lock Count" />
-    <Field type="uint" name="stallCount" label="Stall Count" />
+    <Field type="uint" name="lockCount" label="Lock Count" description="the number of java threads in a critical section when the GC locker is started" />
```

Comment:
I would suggest changing this to:
 
"The number of Java threads in a critical section when the GC locker is started"
"The number of Java threads stalled by the GC locker"

Reply:
changed.